content/js/install-sdk.md

```diff
@@ -125,18 +125,56 @@ cd new-skygear-project
 yo skygear
 ```
 
-### Step 3: Answering the questions for generator
+You will see console output and questions for you that looks like the following:
 
-You will then be prompted a few questions, please make sure that you
-answer these questions correctly:
-- What is your skygear endpoint?
-- What is your skygear API key?
-- Please choose your application name?
-- Overwrite `<some-path>/<some-file>.js`? (confirm to overwrite)
+```
+Just found a `.yo-rc.json` in a parent directory.
+Setting the project root at: /Users/lijiahang/Developer/oursky
+Welcome to skygear!
+? What is your skygear endpoint? (You can find it in portal) https://<your-app-name>.skygeario.com/
+? What is your skygear API key (You can find it in portal) <your-api-key>
+
+I'm all done. Running npm install for you to install the required dependencies. If this fails, try running the command yourself.
+
+     _-----_
+    |       |
+    |--(o)--|   .--------------------------.
+   `---------´  |    Welcome to Yeoman,    |
+    ( _´U`_ )   |   ladies and gentlemen!  |
+    /___A___\   '__________________________'
+     |  ~  |
+   __'.___.'__
+ ´   `  |° ´ Y `
+
+Out of the box I include Webpack and some default React components.
```

Comment:
I think you can keep only the relevant lines (or something like "blah blah blah")? And is there a way you can highlight the user input?